test/k8sT/DatapathConfiguration.go

```diff
@@ -396,7 +395,7 @@ var _ = Describe("K8sDatapathConfig", func() {
 			res.ExpectSuccess()
 			tmpEchoPodPath = strings.Trim(res.Stdout(), "\n")
 			kubectl.ExecMiddle(fmt.Sprintf("sed 's/NODE_WITHOUT_CILIUM/%s/' %s > %s",
-				helpers.GetFirstNodeWithoutCilium(), echoPodPath, tmpEchoPodPath)).ExpectSuccess()
+				helpers.CiliumNamespace, echoPodPath, tmpEchoPodPath)).ExpectSuccess()
```

Comment:
The test failure is because of this line. I'm guessing it was replaced by mistake.

Reply:
Good catch, thanks!